ui/src/app/shared/services/workflows-service.ts

```diff
@@ -241,21 +241,45 @@ export const WorkflowsService = {
                     }
                     throw new Error('no artifact logs are available');
                 }
-
                 return from(requests.get(this.getArtifactLogsPath(workflow, nodeId, container, archived)));
             }),
             mergeMap(r => r.text.split('\n')),
+            filter(x => !!x), //check for empty content
```

Comment:
```suggestion
            filter(x => !!x), // check for empty content
```
nit: still missing a space after the comment slashes, as mentioned [previously](https://github.com/argoproj/argo-workflows/pull/12591#discussion_r1517120807)

Reply:
Oh I had actually overlooked that one.